otto.go

```diff
@@ -238,13 +239,20 @@ type Otto struct {
 	// See "Halting Problem" for more information.
 	Interrupt chan func()
 	runtime   *_runtime
+	console   Console
 }
 
 // New will allocate a new JavaScript runtime
-func New() *Otto {
+func New(options ...Option) *Otto {
 	self := &Otto{
 		runtime: newContext(),
+		console: &console{os.Stdout},
```

Comment:
As this can't ever be configured by the user no need for it.
```suggestion
		console: &console{},
```